pkgs/tools/package-management/cargo-download/default.nix

```diff
@@ -0,0 +1,11 @@
+{ stdenv, lib, fetchgit, darwin, buildPlatform
+, buildRustCrate, defaultCrateOverrides }:
+
+((import ./Cargo.nix { inherit lib buildPlatform buildRustCrate fetchgit; }).cargo_download {}).override {
+  crateOverrides = defaultCrateOverrides // {
+    cargo-download = attrs: {
+      buildInputs = lib.optional stdenv.isDarwin
```

Comment:
I noticed that cargo-edit used a propagatedBuildInput for Security. I wasn't sure if I should be doing the same here. Should I be?

Reply:
Sorry, don't know too much about how the Darwin frameworks work.
If it works for you on a Mac it should be ok.